app/controllers/tmdb_controller.rb

```diff
@@ -15,7 +14,7 @@ def search
 
   def two_movie_search
     if params[:movie_one] && params[:movie_two]
-      @search_results = tmdb_handler_two_movie_search(params[:movie_one], params[:movie_two])
+      @search_results = tmdb_handler_search_common_actors_in_two_movies(params[:movie_one], params[:movie_two])
```

Comment:
This rename isn't relevant to the rest of the work in this PR. I changed it because it was confusing us the other day and renaming it would make that pain go away.